js/components/moter/motebehov/MotebehovInfoForSvar.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+/* eslint-disable max-len */
+export const TEKSTER = {
+    forDuSvarer: {
+        tittel: 'Før du svarer',
+        lagetPlan: 'Har du laget en oppfølgingsplan? Husk å dele den med NAV nå.',
+        ikkeLagetPlan: 'Er oppfølgingsplanen ikke laget? Bruk gjerne den digitale planen du finner når du er logget inn.',
+    },
+    tekstInformasjonInnhold: {
+        lenke: 'Opprett en ny plan.',
+    },
+};
+/* eslint-enable max-len */
+
+export const TekstInformasjonBilde = () => {
```

Comment:
Inline SVG trenger <title>-element for å være ihht UU. Men ideelt sett ville jeg ha lagt bildet i en SVG-fil og brukt img-tag med alt-attributt. Det er mer robust ift. skjermlesere.